test/Microsoft.Web.LibraryManager.Vsix.Test/UI/Models/LibraryIdViewModelTests.cs

```diff
@@ -18,7 +18,17 @@ public class LibraryIdViewModelTests
         private readonly ILibraryCatalog _testCatalog = new LibraryCatalog()
             .AddLibrary(new Library { Name = "test", Version = "1.0" })
             .AddLibrary(new Library { Name = "test", Version = "1.2" })
-            .AddLibrary(new Library { Name = "test", Version = "2.0" });
+            .AddLibrary(new Library { Name = "test", Version = "2.0" })
+            .AddLibrary(new Library { Name = "@types/node", Version = "1.0" })
+            .AddLibrary(new Library { Name = "@types/node", Version = "1.2" })
+            .AddLibrary(new Library { Name = "@types/node", Version = "2.0" })
+
```

Comment:
nit: don't have the empty line.  It breaks the reader from the fluent syntax.

Reply:
Resolved in commit 2